chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -210,7 +210,10 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
 export const endVoiceVideoCallIfOngoing = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>) => {
     let callId = "";
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk = null;
+        if (chatSDK.isVoiceVideoCallingEnabled()) {
```

Comment:
I think all the logic for end chat should be inside this block, if voice is not enabled then next block is irrelevant as well